documentation/src/main/asciidoc/stories/assembly_server_getting_started.adoc

```diff
@@ -3,6 +3,12 @@
 = Getting Started with {brandname} Server
 Quickly set up {brandname} Server and learn the basics.
 
+//Community
+ifdef::community[]
+image:https://raw.githubusercontent.com/infinispan/infinispan.github.io/develop/assets/images/documentation/icon-get-started.png[Get started icon,height=32]
```

Comment:
Shouldn't we use https://infinispan.org/assets/images/documentation/icon-get-started.png ?

Reply:
Thanks @danberindei That's better!